ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/poe.schema.yml

```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  poe:
+    documentation_options:
+      filename: data_model/POE
+    type: dict
+    keys:
+      reboot:
+        type: dict
+        description: Set the global level PoE power behavior for PoE ports when the system is rebooted
+        keys:
+          action:
```

Comment:
Would this create an issue when it comes to non-poe switches though?  Basically, would a non-poe-capable switch still have POE EOS CLI generated for it?

Reply:
I take it back. Looks like there's no issue with setting defaults for the poe values.  However, I believe I'm setting these defaults for documentation purposes on line 3 of [ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/poe.j2](https://github.com/aristanetworks/ansible-avd/pull/2690/files/206c0e7021927e3f4a19f7f4197934d9e0b95fa2#diff-474c47fe975239f18b2e733ee14dd541bd27a6dd719bbe354a519715a4f85ad3).

Not saying this is the best approach but just want to make sure you're aware of it.

Reply:
Ah yeah so basically in documentation it will states that this is the default from EOS but in the config it does not show up. I did not notice indeed. Maybe in that case add in the description that by default on EOS this is what is there as you did in some other places. Thanks for the comment!